model/PFAHandler.php

```diff
@@ -59,6 +59,9 @@
     # set in __construct()
     protected $allowed_domains = false;
 
+    # array with the server list set in __construct()
+    protected $allowed_servers = false;
+
```

Comment:
(sorry for the long delay!)

This is only used in DomainHandler, so it should be defined there. (If it's used by more *Handler classes in the future, PFAHandler would be ok.)